proto/tendermint/abci/types.proto

```diff
@@ -169,26 +169,26 @@ message RequestFinalizeBlock {
 
 message Response {
   oneof value {
-    ResponseException          exception              = 1;
-    ResponseEcho               echo                   = 2;
-    ResponseFlush              flush                  = 3;
-    ResponseInfo               info                   = 4;
-    ResponseInitChain          init_chain             = 5;
-    ResponseQuery              query                  = 6;
-    ResponseBeginBlock         begin_block            = 7 [deprecated = true];
-    ResponseCheckTx            check_tx               = 8;
-    ResponseDeliverTx          deliver_tx             = 9 [deprecated = true];
-    ResponseEndBlock           end_block              = 10 [deprecated = true];
-    ResponseCommit             commit                 = 11;
-    ResponseListSnapshots      list_snapshots         = 12;
-    ResponseOfferSnapshot      offer_snapshot         = 13;
-    ResponseLoadSnapshotChunk  load_snapshot_chunk    = 14;
-    ResponseApplySnapshotChunk apply_snapshot_chunk   = 15;
-    ResponsePrepareProposal    prepare_proposal       = 16;
-    ResponseProcessProposal    process_proposal       = 17;
+    ResponseException           exception             = 1;
+    ResponseEcho                echo                  = 2;
+    ResponseFlush               flush                 = 3;
+    ResponseInfo                info                  = 4;
+    ResponseInitChain           init_chain            = 5;
+    ResponseQuery               query                 = 6;
+    ResponseBeginBlock          begin_block           = 7 [deprecated = true];
```

Comment:
I know we've discussed this a few times, but I'm not sure what the outcome was. 

I think we decided that we're keeping the OG ABCI types here so that the SDK can play bridge for existing ABCi applications that have not yet been updated to use the ABCI++ requests. If that's correct (which would make sense), we should double-check that we have not changed any of the existing type tags on the request and response `oneof` wrappers. Otherwise, an existing out-of-process application will get the wrong tag numbers for the existing messages and is likely to be confused.

Reply:
cc @sergio-mena

Reply:
The tag numbers have been kept consistent with the old version.
As for what to do with these, it was indeed discussed [here](https://github.com/tendermint/spec/pull/392#discussion_r798163606), where I proposed to follow @creachadair's proposal 3).
My view hasn't changed on this: once this PR is in (meaning: the bulk of ABCI++ implementation is in) we should remove this entirely to avoid confusion (Apps might think we still support the old API, which is not the case).

Reply:
> The tag numbers have been kept consistent with the old version.

Ok, great, in that case I have no concerns. I just wasn't sure because of how the new ones are mixed in. ✔️